app/assets/javascripts/components/sidebar_item.es6.jsx

```diff
@@ -1,28 +1,38 @@
 class SidebarItem extends React.Component {
 
-	get styles() {
+  static get propTypes() {
+    return {
+      label: React.PropTypes.string,
+      icon: React.PropTypes.string,
```

Comment:
Here, we can add `.isRequired` to the end of each `React.PropTypes.string` because we're going to make both the `label` and the `icon` props required.